src/views/ScaleManagerDemo.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="scaleManager__container">
+    <div class="scaleManager__wrapper">
+      <a class="scaleManager__readme"  href="https://github.com/SpringRoll/SpringRoll/tree/develop/src/scale-manager">Scale Manager README</a>
+      <iframe id="scaleManager-demo" allow="fullscreen" class="scaleManager" :src="`${publicPath}SafeScaleManagerDemo`" frameborder="0" />
+      <v-btn @click="fullScreen" block color="primary" class="scaleManager__button --fullScreen --capital font-16 font-semi-bold">Full Screen</v-btn>
+      <a class="scaleManager__source"  href="https://github.com/SpringRoll/springroll-io-safe-scale-manager-demo/tree/feature/safe-scale-manager-demo">Source Code for the demo game</a>
+    </div>
+    <div class="scaleManager__events">
+      <h2 class="scaleManager__header">Scaling Options</h2>
+      <v-form
+      ref="scaleForm"
+      v-model="scaleValid"
+      >
+      <template v-for="value in scaleValues">
+        <!-- <label class="scaleManager__label" :for="value.label" :key="value.label">{{ value.label }}: -->
+        <v-text-field
```

Comment:
I'm seeing some weird behavior in my browser when the element gains focus. The label underneath shows up which causes text to reflow in a fairly jarring way. I think there's some missing `max-width` somewhere causing the labels underneath to example this container.